cea/import_export/import_from_rhino_gh.py

```
@@ -40,18 +39,21 @@ def exec_import_csv_from_rhino(locator):
+    dh_edges_csv_path = os.path.join(input_path, 'dh_edges_in.csv')
+    dh_nodes_csv_path = os.path.join(input_path, 'dh_nodes_in.csv')
+    dc_edges_csv_path = os.path.join(input_path, 'dc_edges_in.csv')
+    dc_nodes_csv_path = os.path.join(input_path, 'dc_nodes_in.csv')
 
     # Create the CEA Directory for the new scenario
     input_path = locator.get_input_folder()
     building_geometry_path = locator.get_building_geometry_folder()
-    building_properties_path = locator.get_building_properties_folder()
     networks_path = locator.get_networks_folder()
     trees_path = locator.get_tree_geometry_folder()
+    thermal_network_path = locator.get_thermal_network_folder()
```

Ruff (0.8.2)

Local variable `thermal_network_path` is assigned to but never used

Remove assignment to unused variable `thermal_network_path`

(F841)